toolz/itertoolz.py

```diff
@@ -15,6 +14,9 @@
            'join', 'tail', 'diff', 'topk', 'peek')
 
 
+_no_default = object()  # sentinel
```

Comment:
Would it suffice if `cytoolz` tried to import this sentinel value from `toolz` if `toolz` is installed?